jacoco-aggregate-report/pom.xml

```diff
@@ -5,7 +5,7 @@
 	<parent>
 		<artifactId>uniprot-rest-api</artifactId>
 		<groupId>org.uniprot</groupId>
-		<version>1.0.26-SNAPSHOT</version>
+		<version>1.0.25.2</version>
```

Comment:
Can't we use ${project.parent.version} for these places as well (including other similar places here)?

Reply:
This is updated by release script. I updated back to 1.0.26-SNAPSHOT, this way we can merge it to master